doc/command_ref.rst

```diff
@@ -1223,6 +1223,9 @@ Repoquery Command
     Searches available DNF repositories for selected packages and displays the requested information about them. It
     is an equivalent of ``rpm -q`` for remote repositories.
 
+``dnf [options] repoquery --groupmember <package-spec>...``
+    Query what group the <package-spec> belong to.
```

Comment:
Could you change the text to:
`List groups that contain <package-spec>.`
I think it's easier to read.

Reply:
No problem. It's done.